src/type/validate.js

```diff
@@ -314,94 +317,110 @@ function validateFields(
   }
 }
 
-function validateObjectInterfaces(
+function validateInterfaces(
   context: SchemaValidationContext,
-  object: GraphQLObjectType,
+  implementing: GraphQLObjectType | GraphQLInterfaceType,
```

Comment:
Can you please change the name to `type` similar to `validateFields`:
https://github.com/graphql/graphql-js/blob/5873fd26f60e88c473dea678a7723888f766658b/src/type/validate.js#L259-L262